### PR TITLE
Trip command

### PR DESCRIPTION
**Pull request by markmcwong**
- Show pools on right panel
- `Trip` command for filtering right panel 
- Fix case-sensitivity for creation of TripDay
- `Pool` command now add pools to AddressBook

### CODE REVIEW

**Review by JoelHo — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -121,12 +121,12 @@ public static Tag parseTag(String tag) throws ParseException {
     /**
      * Parses a {@code String tripDay} into a {@code TripDay}.
      * Leading and trailing whitespaces will be trimmed.
-     *
+     * and converted to uppercase for case-insensitive enum instantiation of {@code DayOfWeek}
```
Comment (chewterence) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: perhaps you could add a fullstop here

File: src/main/java/seedu/address/logic/parser/TripCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+/**
+ * Parses input arguments and creates a new FindCommand object
+ */
+public class TripCommandParser implements Parser<TripCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindCommand
+     * and returns a FindCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```
Comment (chewterence) [inline src/main/java/seedu/address/logic/parser/TripCommandParser.java]: fullstop here too

File: src/main/java/seedu/address/logic/parser/TripCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        assert(presentPrefixes.size() == 1);
+
+        Prefix specifiedPrefix = presentPrefixes.get(0);
+        List<String> keywords = parseValue(argMultimap, specifiedPrefix);
+
+        return new TripCommand(parsePredicate(specifiedPrefix, keywords));
+    }
+
+    /**
+     * Returns true if only one of the prefixes are provided
```
Comment (chewterence) [inline src/main/java/seedu/address/logic/parser/TripCommandParser.java]: perhaps a fullstop here too

**Review by chewterence — COMMENTED** [review]
please check my comments, thank you

**Pull request merged**